main.go

```diff
@@ -54,6 +54,7 @@ var (
 
 	samplesQueueCapacity = flag.Int("storage.incoming-samples-queue-capacity", 64*1024, "The capacity of the queue of samples to be stored. Note that each slot in the queue takes a whole slice of samples whose size depends on details of the scrape process.")
 
+	chunkType       = flag.Int("storage.local.chunk-type", 1, "Which chunk encoding version to use. Currently supported is 0 and 1.")
```

Comment:
Definitely yes. Three reasons:

- Double-delta chunks are fundamentally computationally more expensive. With predominantly random data, there is no gain in space efficiency to compensate. This effect can be seen in the benchmarks. It's pretty relevant. Users should be able to choose. Double-delta chunks as default is sane, though, and users who are not interested are not bothered. It's an optional flag after all.

- Users might be interested to keep their storage compatible with older versions. (To be able to roll back if a new version introduces a bug.)

- Finally, we will certainly introduce more chunk types in the future, where the choice gets more interesting.